auth-web/src/models/business.ts

```diff
@@ -61,7 +61,8 @@ export interface NameRequest {
     state?: string,
     applicantEmail?: string,
     applicantPhone?: string,
-    enableIncorporation?: boolean
+    enableIncorporation?: boolean,
+    target?: NrTargetTypes
```

Comment:
Why not update and use the shared interface for a Name Request?

Reply:
This interface is used for instances beyond just the response from the NR, so the optional properties might not be exactly the same. 
Will have a look though and use if able!

Reply:
I think TS allows the use of Partial interfaces, which may allow you to use a shared interface and add in your own properties... but I haven't used that yet.

Reply:
Or this: https://www.typescripttutorial.net/typescript-tutorial/typescript-extend-interface/

Reply:
The discrepancy here is this interface is to define the Namex Response object (partially) not the NameRequest used by our UI's for Filings/Corrections etc. 

I could add a Namex Response interface in the BCRS repo and import it back.

Reply:
🤷‍♂️

Reply:
OK, I think you should not add a one-time Namex Response interface to the shared repo.

If you can't find a way to leverage the usual NR contents (interface) with this interface (which is unfortunately named the same as the other NR interfaces) then just leave it, I guess. I was hoping to introduce some consistency / schema-like thing but oh well.

Reply:
100% i know where you're coming from. It's caused me a little angst trying to sort this out because i do agree!

That being said, there is some interfaces overlap here (as well as other ui's) which require a bit more sorting out. 
What i mean is removing any localized NameRequest related Interfaces and just relying on the shared Interfaces, but would be a refactor stretching a little farther outside the scope here.

Reply:
So what do you want to do?

Reply:
If we feel strongly enough about this, we should make a ticket.
I still feel like a guest in this repo, so i'm less inclined to refactor where it isn't absolutely required.

Reply:
> I still feel like a guest in this repo
Haha, me too!

> If we feel strongly enough about this, we should make a ticket.
Sure, make a ticket. They can always decide to not do it (or make it low priority).

Reply:
Ticket is made! 
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/8794